### PR TITLE
Basic framework code

### PR DESCRIPTION
**Pull request by xxwjj**
Basic framework code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# anomaly-detection
-Repo for anomaly detection
+# orchestration
+Repo for automation and orchestration
```
Comment (xing-yang) [inline README.md]: Copy and paste error.  Remove these changes.  The original description is correct.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+setup(
+    name="anomaly_detection",
+    version="1.0",
+    url="https://opensds.io",
+    description="A service catalog manager for OpenSDS",
```
Comment (xing-yang) [inline setup.py]: An anomaly detection project for OpenSDS

**Review by xing-yang — COMMENTED** [review]
Rename folders:
s/dataparser/data_parser
s/datagenerator/data_generator

File: anomaly_detection/server.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        self._init_server()
+
+    def _init_logging(self):
+        pass
+
+    def _init_server(self):
+        self.app.url_map.strict_slashes = False
+
+        # register router
+        # self.app.register_blueprint(class_name)
```
Comment (xing-yang) [inline anomaly_detection/server.py]: Remove commented out line.

File: README.md (hunk 1 above)
Comment (xxwjj) [reply README.md]: done

File: anomaly_detection/server.py (hunk 3 above)
Comment (xxwjj) [reply anomaly_detection/server.py]: done

File: setup.py (hunk 2 above)
Comment (xxwjj) [reply setup.py]: done

**Review by xing-yang — APPROVED** [review]
lgtm

**Pull request merged**